CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/utilities/MarkdownTree.java

```diff
@@ -363,13 +362,15 @@ private void parseString(final MarkdownNode currentNode, String text) {
                         // and the next line does not start with the same amount of tabs or it does start with the same amount of tabs
                         // and there is a new tab right after that OR there are no tabs on the list item and the next line contains a tab
                         // then change end index of the \n for this next line
-                        while (endIndex != -1 && ((!tabString.toString().isEmpty()
-                                && (text.indexOf(tabString.toString(), endIndex + 1) != endIndex + 1
-                                || (text.indexOf(tabString.toString(), endIndex + 1) == endIndex + 1
+                        String tabs = tabString.toString();
+                        while (endIndex != -1 && ((StringUtils.isEmpty(tabs)
+                                && (text.indexOf(tabs, endIndex + 1) != endIndex + 1
+                                || (text.indexOf(tabs, endIndex + 1) == endIndex + 1
                                 && text.indexOf("\t", endIndex + currentNode.getTabs() + 1) == endIndex + currentNode.getTabs() + 1)) || text.indexOf("\n", endIndex + 1) == endIndex + 1)
-                                || (tabString.toString().isEmpty()
+                                || (StringUtils.isEmpty(tabs)
                                 && text.indexOf("\t", endIndex + 1) == endIndex + 1 || text.indexOf("\n", endIndex + 1) == endIndex + 1))) {
                             endIndex = text.indexOf("\n", endIndex + 1);
+                            tabs = tabString.toString();
```

Comment:
I may have missed something, but it doesn't seem like this value is different to the previous instantiation (as `tabString` doesn't change in the while loop). Therefore you should be able to remove this one and make the first instantiation final